test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs

```diff
@@ -20,7 +16,7 @@ public static HttpContextAccessor CreateHttpContextAccessor(RequestTelemetry req
             var request = new DefaultHttpContext().Request;
             request.Method = "GET";
             request.Path = new PathString("/Test");
-            var contextAccessor = new HttpContextAccessor() { HttpContext = request.HttpContext };
+            var contextAccessor = new HttpContextAccessor { HttpContext = request.HttpContext };
```

Comment:
You do not need them when you define properties inline.

---
In reply to: [70717232](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/244#discussion_r70717232) [](ancestors = 70717232)